src/main/java/org/jenkinsci/plugins/casc/plugins/PluginManagerConfigurator.java

```diff
@@ -104,6 +104,7 @@ public PluginManager configure(CNode config) throws ConfiguratorException {
         }
 
         File shrinkwrap = new File("./plugins.txt");
```

Comment:
Storing it in a relative path looks dangerous. Why not just a fixed path in JENKINS_HOME?

Reply:
I'd want to do that in a seperate pull request. For now im just interested to see what path is resolved inside my container.